js/extension/components/FeatureReports.jsx

```diff
@@ -125,6 +139,7 @@ class FeatureReports extends React.Component {
                                 onChange={log("changed")}
                                 onSubmit={this.props.postReport}
                                 onError={log("errors")}
+                                FieldTemplate={fieldTemplate}
```

Comment:
This being here impacts the way the form is displayed :

without it :
![without_fieldTemplate](https://user-images.githubusercontent.com/78360657/118688339-42810b80-b806-11eb-967d-5b1984157bcb.png)

with it :
![with_FieldTemplate](https://user-images.githubusercontent.com/78360657/118688407-5298eb00-b806-11eb-804f-3ade630117de.png)